lib/run.go

```diff
@@ -171,13 +171,14 @@ func findCmdInPath(pathlist []string, cmd, prefix string) (string, error) {
 	}
 
 	for _, p := range pathlist {
-		ex, err := util.Exists(path.Join(prefix, p, cmd))
```

Comment:
SGTM

On Tue, Nov 3, 2015, 18:35 Derek Gonyeo <notifications@github.com> wrote:

> In lib/run.go
> <https://github.com/appc/acbuild/pull/91#discussion_r43779582>:
>
> > @@ -171,13 +171,14 @@ func findCmdInPath(pathlist []string, cmd, prefix string) (string, error) {
> >  	}
> >
> >  	for _, p := range pathlist {
> > -		ex, err := util.Exists(path.Join(prefix, p, cmd))
>
> I'm actually thinking about just removing the util.Exists function. It
> doesn't really save much code anywhere, since each time I use it I still
> need to specify what to do if the file doesn't exist, what to do if there
> was an error, and what to do if the file exists. It doesn't simplify any of
> the logic.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/appc/acbuild/pull/91/files#r43779582>.
>